source/_references/stacks.md

```diff
@@ -73,6 +73,101 @@ print_r($stack);
 
 Note: The name "dynamic" (used for dynamic rendering) and names starting with "_" are reserved and can't be chosen as stack names.
 
+### Updating stacks
+
+Please read this carefully, if you want to overwrite existing stacks with new options/operations and the same name, since it may not work like you'd expect.
```

Comment:
no, i meant the same name, but wrote that clearer now (replacing overwriting with updating makes it clearer)